lib/question.js

```diff
@@ -368,8 +369,10 @@ module.exports = {
             },
             (callback) => { // if workspace, encode graded files into _files
                 if (question.workspace_image !== null) {
-                    logger.info(`... reading workspace-${workspace_id}-graded-files.zip`);
-                    const zip = new admZip(`workspace-${workspace_id}-graded-files.zip`);
+                    const zipPrefix = config.workspaceGradedFilesMainDirectory;
+                    const zipName = `${zipPrefix}/workspace-${workspace_id}-graded-files.zip`
```

Comment:
I think we should make these names unique in case we end up with overlapping grading jobs. Either add a timestamp or a UUID.